spec/system/casa_org/edit_spec.rb

```diff
@@ -75,6 +75,24 @@
     message = find("#casa_org_twilio_phone_number").native.attribute("validationMessage")
     expect(message).to eq "Please fill out this field."
   end
+
+  it "displays learning hour topic in learning hour when enabled", js: true do
+    organization = create(:casa_org, learning_topic_active: true)
+    volunteer = create(:volunteer, casa_org: organization)
+
+    sign_in volunteer
+    visit new_volunteer_learning_hour_path(volunteer)
+    expect(page).to have_text("Learning Topic")
+  end
+
+  it "does not display learning hour topic in learning hour when disabled", js: true do
+    organization = create(:casa_org)
+    volunteer = create(:volunteer, casa_org: organization)
+
+    sign_in volunteer
+    visit new_volunteer_learning_hour_path(volunteer)
+    expect(page).to_not have_text("Learning Topic")
+  end
```

Comment:
Since these are visiting the `new_volunteer_learning_hour_path`, could you move these specs to `spec/system/volunteers/learning_hours/new_spec.rb`